manifests/operator.yaml

```diff
@@ -202,6 +202,7 @@ spec:
       - name: operator
         image: gke.gcr.io/prometheus-engine/operator:v0.4.1-gke.0
         args:
+        - "--operator-namespace=gmp-system"
         - "--public-namespace=gmp-public"
         - "--priority-class=gmp-critical"
```

Comment:
I don't think this is still supported, right?

Reply:
It's still [supported](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/freinartz/yamlcollector/cmd/operator/main.go#L60-L61) and relevant for future templating - i.e. deploying all resources in a particular namespace.